reference/configuration/framework.rst

```diff
@@ -1187,7 +1189,94 @@ individually for each asset package:
 
 .. note::
 
-    This parameter cannot be set at the same time as ``version``.
+    This parameter cannot be set at the same time as ``version`` or ``json_manifest_path``.
+
+.. _reference-assets-json-manifest-path:
+
+json_manifest_path
+..................
+
+**type**: ``string`` **default**: ``null``
+
+.. versionadded:: 3.3
+
+    The ``json_manifest_path`` option was introduced in Symfony 3.3.
+
+The file path to a `manifest.json` file containing an associative array of asset
+names and their respective compiled names. A common cache-busting tecnique using
+a `manifest.json` file works by writing out assets with a "hash" or "chunk hash"
```

Comment:
I think we can remove `or "chunk hash"` - it may just confuse people - the important thing is that it's some hash in the filename